config/hg19.yml

```diff
@@ -47,7 +47,7 @@ statistics:
   programPath: bystro-stats
   refTrackField: ref
   siteTypeField: refSeq.siteType
-temp_dir: null
+temp_dir: /mnt/annotator/tmp
```

Comment:
It turns out that for big_daly, and presumably similarly sized datasets, writing to local ssd tmp is faster, and because bystro writes data so efficiently, even for big_daly, which is 2.7TB compressed, we write only ~ 120GB compressed, so tmp storage can be fairly modest.